src/test/java/org/apache/commons/collections4/CollectionUtilsTest.java

```diff
@@ -1472,6 +1472,101 @@ public void testRetainAll() {
         } // this is what we want
     }
 
+    @Test
+    public void testRemoveRange() {
+        List<Integer> list = new ArrayList<Integer>();
```

Comment:
FYI: You don't need to over specify generics in Java 8 anymore, so:
```
List<Integer> list = new ArrayList<Integer>();
```
can simply be:
```
List<Integer> list = new ArrayList<>();
```

Reply:
Also please rebase on master, then I'll take another look. TY!